sdk/inc/azure/core/az_mqtt5_rpc_server.h

```diff
@@ -164,6 +164,14 @@ typedef struct
    * @brief The content type of the request.
    */
   az_span content_type;
+  /**
+   * @brief The service id of the request.
+   */
+  az_span service_id;
+  /**
+   * @brief The command name of the request.
+   */
+  az_span command_name;
```

Comment:
@RLeclair was there a reason we didn't include the command name in this before?

Reply:
This should have been included here, the change is correct